example-wasm/src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
```

Comment:
Is this something we need in an example?

Reply:
Depends on if users want to use that to measure performance. Like you're saying, unsure if we want to keep it as part of the example app or not. Users could always add it themselves as well.

Reply:
Sure, does it come with CRA?

Reply:
Yup!